sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/StatusTests.cs

```diff
@@ -48,8 +43,7 @@ public void ValidateStatusForAzureMonitorTrace(string activityStatusCode, string
                 parentContext: new ActivityContext(ActivityTraceId.CreateRandom(), ActivitySpanId.CreateRandom(), ActivityTraceFlags.Recorded),
                 startTime: DateTime.UtcNow);
 
-            activity.SetTag("otel.status_code", activityStatusCode);
-            activity.SetTag("otel.status_description", activityStatusDescription);
+            activity.SetStatus(activityStatusCode, activityStatusDescription);
```

Comment:
do we use description anywhere in the tests or product?

Reply:
No we do not - Test needed some cleanup. Have removed it now.